vms/avm/vm.go

```diff
@@ -478,31 +478,20 @@ func (vm *VM) ParseTx(_ context.Context, bytes []byte) (snowstorm.Tx, error) {
  */
 
 // IssueTx attempts to send a transaction to consensus.
-// If onDecide is specified, the function will be called when the transaction is
-// either accepted or rejected with the appropriate status. This function will
-// go out of scope when the transaction is removed from memory.
-func (vm *VM) IssueTx(b []byte) (ids.ID, error) {
-	if !vm.bootstrapped || vm.Builder == nil {
-		return ids.ID{}, errBootstrapping
-	}
```

Comment:
This isn't needed anymore. We only allow API traffic after bootstrapping has finished, and we are guaranteed to have Linearize called during bootstrapping.